src/main/plugins/org.dita.pdf2/xsl/fo/hazard-domain.xsl

```diff
@@ -0,0 +1,286 @@
+      </fo:table-body>
+    </fo:table>
+  </xsl:template>
+
+  <xsl:template name="hazardStatementHead">
+    <xsl:choose>
+      <xsl:when test="@type eq 'note'">
+        <fo:block xsl:use-attribute-sets="hazardstatement-note-head" >
+          <xsl:call-template name="hazardStatementHeadContent" />
+        </fo:block>
+      </xsl:when>
+
+      <xsl:when test="@type eq 'tip'">
+        <fo:block xsl:use-attribute-sets="hazardstatement-tip-head">
+            <xsl:call-template name="hazardStatementHeadContent" />
+        </fo:block>
+      </xsl:when>
+
+      <xsl:when test="@type eq 'fastpath'">
+        <fo:block xsl:use-attribute-sets="hazardstatement-fastpath-head">
+            <xsl:call-template name="hazardStatementHeadContent" />
+        </fo:block>
+      </xsl:when>
+
+      <xsl:when test="@type eq 'restriction'">
+        <fo:block xsl:use-attribute-sets="hazardstatement-restriction-head">
+            <xsl:call-template name="hazardStatementHeadContent" />
+        </fo:block>
+      </xsl:when>
+
+      <xsl:when test="@type eq 'important'">
+        <fo:block xsl:use-attribute-sets="hazardstatement-important-head">
+            <xsl:call-template name="hazardStatementHeadContent" />
+        </fo:block>
+      </xsl:when>
+
+      <xsl:when test="@type eq 'remember'">
+        <fo:block xsl:use-attribute-sets="hazardstatement-remember-head">
+            <xsl:call-template name="hazardStatementHeadContent" />
+        </fo:block>
+      </xsl:when>
+
+      <xsl:when test="@type eq 'attention'">
+        <fo:block xsl:use-attribute-sets="hazardstatement-attention-head">
+            <xsl:call-template name="hazardStatementHeadContent" />
+        </fo:block>
+      </xsl:when>
+
+      <xsl:when test="@type eq 'caution'">
+        <fo:block xsl:use-attribute-sets="hazardstatement-caution-head">
+            <xsl:call-template name="hazardStatementHeadContent" />
+        </fo:block>
+      </xsl:when>
+
+      <xsl:when test="@type eq 'notice'">
+        <fo:block xsl:use-attribute-sets="hazardstatement-notice-head">
+            <xsl:call-template name="hazardStatementHeadContent" />
+        </fo:block>
+      </xsl:when>
+
+      <xsl:when test="@type eq 'danger'">
+        <fo:block xsl:use-attribute-sets="hazardstatement-danger-head">
+            <xsl:call-template name="hazardStatementHeadContent" />
+        </fo:block>
+      </xsl:when>
+
+      <xsl:when test="@type eq 'warning'">
+        <fo:block xsl:use-attribute-sets="hazardstatement-warning-head">
+            <xsl:call-template name="hazardStatementHeadContent" />
+        </fo:block>
+      </xsl:when>
+
+      <xsl:otherwise>
+        <fo:block xsl:use-attribute-sets="hazardstatement-other-head">
+            <xsl:call-template name="hazardStatementHeadContent" />
+        </fo:block>
+      </xsl:otherwise>
+    </xsl:choose>
+  </xsl:template>
+  
+  <xsl:template match="*[contains(@class,' hazard-d/hazardstatement ')]" mode="setIconImagePath">

```

Comment:
This looks like an exact duplicate of the current `<note>` processing, except with a new mode - I'm not sure why we would not use the existing lookup? Anybody can still override the original, but overriding core code inside the core code, without any change, does not seem to buy anything.